lib/helpers/git_helper.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require 'git'
+require 'fileutils'
+
+def get_config (config_file)
+  json_data = []
+  begin
+    json_file = File.read(config_file)
+    json_data = JSON.parse(json_file)
+  rescue Exception => e
+    puts e.message
```

Comment:
Dont `puts` errors use `STDERR.puts` instead. `puts` sends to STOUT which should not be used for errors.